src/qtacrylichelper_global.h

```diff
@@ -27,19 +27,19 @@
 #include <QtCore/qglobal.h>
 
 #ifndef QTACRYLICHELPER_API
-#ifdef QTACRYLICHELPER_STATIC
-#define QTACRYLICHELPER_API
-#else
-#ifdef QTACRYLICHELPER_BUILD_LIBRARY
-#define QTACRYLICHELPER_API Q_DECL_EXPORT
-#else
-#define QTACRYLICHELPER_API Q_DECL_IMPORT
-#endif
-#endif
+    #ifdef QTACRYLICHELPER_STATIC
+        #define QTACRYLICHELPER_API
+    #else
+        #ifdef QTACRYLICHELPER_BUILD_LIBRARY
+            #define QTACRYLICHELPER_API Q_DECL_EXPORT
+        #else
+            #define QTACRYLICHELPER_API Q_DECL_EXPORT // Q_DECL_IMPORT
```

Comment:
Of course deleting the following two lines directly will also fail to compile. Although it is a bit strange.